envoy/Dockerfile

```diff
@@ -14,6 +14,7 @@
 
 FROM envoyproxy/envoy:v1.28.0
 
-COPY envoy/envoy.yaml /etc/envoy/envoy.yaml
+COPY envoy.yaml /etc/envoy/envoy.yaml
 
-CMD /usr/local/bin/envoy -c /etc/envoy/envoy.yaml -l trace --log-path /tmp/envoy_info.log
+ENTRYPOINT [ "/usr/local/bin/envoy" ]
+CMD [ "-c /etc/envoy/envoy.yaml", "-l trace", "--log-path /tmp/envoy_info.log" ]
```

Comment:
Why not splitting all parameter ?

["-c", "/etc/envoy/envoy.yaml", "-l", "trace", "--log-path", "/tmp/envoy_info.log"]

Reply:
I copied it from the official grpc-web repo, 🥲 but you are right, we should split all the paraterms.